doc/aerospike.rst

```diff
@@ -362,6 +362,11 @@ Only the `hosts` key is required; the rest of the keys are optional.
             (Optional) A defined user with roles in the cluster. See :meth:`admin_create_user`.
         * **password** (:class:`str`)
             (Optional) The password will be hashed by the client using bcrypt.
+        * **config_provider** (:class:`aerospike.ConfigProvider`)
+            (Optional) Dynamic configuration provider.
+
+            An alternate way to enable dynamic config is to set environment variable ``AEROSPIKE_CLIENT_CONFIG_URL``
+            to the path of the config file before running the application.
```

Comment:
Is `AEROSPIKE_CLIENT_CONFIG_URL` a file path or a web url? The name makes me think it's a web url but the description says path

Reply:
It's a file path. Like `/home/julian/dyn_config.yml` for example